internal/provider/resource_gitlab_branch_protection.go

```diff
@@ -174,6 +183,10 @@ func resourceGitlabBranchProtectionRead(ctx context.Context, d *schema.ResourceD
 		}
 	}
 
+	if err := d.Set("allow_force_push", pb.AllowForcePush); err != nil {
```

Comment:
NIT: we usually don't implement explicit error checking when setting primitive types (they are caught by the compiler I guess) ... but doesn't hurt either :)

Reply:
I wasn't sure the right approach.  I saw it being done with `code_owner_approval_required` so figured it shouldn't hurt.  If we don't want it done, I can exclude that in the future.